apps/src/lib/kits/maker/ui/SetupInstructions.jsx

```diff
@@ -98,7 +98,6 @@ class Support extends React.Component {
     return (
       <div>
         <h2>{i18n.support()}</h2>
-        <SafeMarkdown markdown={i18n.debugMakerToolkit()} />
```

Comment:
If this string is not used anymore, it should be deprecated from [apps/i18n/common/en_us.json](https://github.com/code-dot-org/code-dot-org/blob/72705dbc245a5adc8ea2c58d3cc9cc88de893ab5/apps/i18n/common/en_us.json#L781)